README.md

```diff
@@ -105,13 +105,14 @@ Note that when using insecure context StreamSaver will navigate to the download
 
 # Configuration
 
+By default, `https://jimmywarting.github.io/StreamSaver.js/mitm.html?version=2.0.0` will be used to create the iframe in secure context. But it could be unavailable when using Firefox or an environment where `github.io` can not be accessed, then you need to config a custom `mitm.html` by yourself.
+
```

Comment:
I do not believe it is a specific Firefox issue?

I think it's more about 3th party cookie restrictions

you see, if you disable 3th party cookies in chrome or in safari then jimmywarting.github.io/StreamSaver.js/mitm.html won't work either...

It's should say something like: 
> StreamSaver uses a 3th party service to install a service worker on an external domain (hidden in a iframe). This can sometimes be restricted by disabling 3th party cookies in your browser. To get around this issue it's best to self-host the service worker and replace `streamSaver.mitm` to your own domain.

Toughts?

Reply:
It seems #242 is not mentioned in the new sentence?

Reply:
unrelated, but... My https://github.com/jimmywarting/native-file-system-adapter works a bit differently... same concept doe... but it communicates with service workers directly without a mitm or any hidden iframes or will use the native file system access `window.showSaveFilePicker`

Reply:
> It seems #242 is not mentioned in the new sentence?

???

fyi, it could also be a restriction with [CSP](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP)... 
should definitely put up a guide of how to set it up more properly, a setting for communicating with service worker directly instead of via a iframe could also be preferred.

Reply:
I mean my words `or an environment where `github.io` can not be accessed`

Reply:
> a setting for communicating with service worker directly instead of via a iframe could also be preferred.

I didn't know we can skip mimt even. 😂

Reply:
> will use the native file system access window.showSaveFilePicker

Yeah, I'm also doing this myself.